domains/0xliumingedwin.json

```
@@ -0,0 +1,10 @@
+{
+  "owner": {
+    "username": "liumingedwin",
+    "email": "[email protected]"
```

invalid email, if you dont want to share you are free to remove this
```suggestion
    "username": "liumingedwin"
```